### ISSUE TITLE
Installs but crashes upon start.

### ISSUE THREAD

**Issue by rkoot** [RvE54/RandoMice#1]
Version 1.09 crashes hard on Windows 2012R2 x64 Terminal server. (fully up to date).
Detail:

Problem Event Name:	CLR20r3
  Problem Signature 01:	RandoMice.exe
  Problem Signature 02:	1.0.9.0
  Problem Signature 03:	5e8f65cd
  Problem Signature 04:	mscorlib
  Problem Signature 05:	4.8.4180.0
  Problem Signature 06:	5e7d20d3
  Problem Signature 07:	2d74
  Problem Signature 08:	2f
  Problem Signature 09:	System.ArgumentException
  OS Version:	6.3.9600.2.0.0.144.8
  Locale ID:	1043
  Additional Information 1:	ac0a
  Additional Information 2:	ac0a7876a06b285aa9ed586afa5729c2
  Additional Information 3:	c2ac
  Additional Information 4:	c2acfe8a0d1269cce1b28500c376f6bd

**Comment by rkoot** [comment]
additional information:
```
- <Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
- <System>
  <Provider Name=".NET Runtime" /> 
  <EventID Qualifiers="0">1026</EventID> 
  <Level>2</Level> 
  <Task>0</Task> 
  <Keywords>0x80000000000000</Keywords> 
  <TimeCreated SystemTime="2020-07-10T08:45:15.000000000Z" /> 
  <EventRecordID>1402928</EventRecordID> 
  <Channel>Application</Channel> 
  <Computer>XXXXXXXXX</Computer> 
  <Security /> 
  </System>
- <EventData>
  <Data>Application: RandoMice.exe Framework Version: v4.0.30319 Description: The process was terminated due to an unhandled exception. Exception Info: System.Globalization.CultureNotFoundException at System.Globalization.CultureInfo.GetCultureInfo(System.String) at System.Resources.ManifestBasedResourceGroveler.GetNeutralResourcesLanguage(System.Reflection.Assembly, System.Resources.UltimateResourceFallbackLocation ByRef) Exception Info: System.ArgumentException at System.Resources.ManifestBasedResourceGroveler.GetNeutralResourcesLanguage(System.Reflection.Assembly, System.Resources.UltimateResourceFallbackLocation ByRef) at System.Resources.ResourceManager.CommonAssemblyInit() at System.Resources.ResourceManager..ctor(System.Type) at System.ComponentModel.ComponentResourceManager..ctor(System.Type) at RvE_RandoMice.MainForm.InitializeComponent() at RvE_RandoMice.MainForm..ctor(System.String[]) at RvE_RandoMice.Program.Main(System.String[])</Data> 
  </EventData>
  </Event>

- <Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
- <System>
  <Provider Name="Application Error" /> 
  <EventID Qualifiers="0">1000</EventID> 
  <Level>2</Level> 
  <Task>100</Task> 
  <Keywords>0x80000000000000</Keywords> 
  <TimeCreated SystemTime="2020-07-10T08:45:15.000000000Z" /> 
  <EventRecordID>1402929</EventRecordID> 
  <Channel>Application</Channel> 
  <Computer>XXXXXXXX</Computer> 
  <Security /> 
  </System>
- <EventData>
  <Data>RandoMice.exe</Data> 
  <Data>1.0.9.0</Data> 
  <Data>5e8f65cd</Data> 
  <Data>KERNELBASE.dll</Data> 
  <Data>6.3.9600.19724</Data> 
  <Data>5ec5262a</Data> 
  <Data>e0434352</Data> 
  <Data>0000000000007afc</Data> 
  <Data>1d98c</Data> 
  <Data>01d656966da1ee41</Data> 
  <Data>C:\Program Files\LUMC\RandoMice\RandoMice.exe</Data> 
  <Data>C:\Windows\system32\KERNELBASE.dll</Data> 
  <Data>ac4d841b-c289-11ea-8110-f2ba4d734568</Data> 
  <Data /> 
  <Data /> 
  </EventData>
  </Event>
```

**Comment by RvE54** [comment]
Hi rkoot, thank you for posting this issue here.
We have been in contact about this issue via e-mail and we have located the source of the error.
The fix will be implemented in v1.1.0.

For now, I have e-mailed you a version of the software that you can use until v1.1.0 is released.
